api/account/tests/features/steps/pantacor_update.py

```diff
@@ -27,7 +27,7 @@
 from selene.testing.device import add_device, add_pantacor_config
 
 
-@given("a device using Pantacor with manual updates enabled")
+@given("a device using Pantacor for continuous delivery")
```

Comment:
should the docstring for this function also be updated?

Reply:
I don't think the docstring is inaccurate.

Reply:
oh I took "continuous delivery" to mean that auto-updates were enabled